src/amo/components/AddonReviewListItem/styles.scss

```diff
@@ -0,0 +1,41 @@
+@import "~amo/css/inc/vars";
+
+.AddonReviewListItem {
+  word-wrap: break-word;
+
+  > h3 {
```

Comment:
I know these are from a previous patch but if it'd be possible to label this with a `className` and select them directly instead that'd be a bit less fragile. I've liked us not nesting selectors, especially direct selectors like this. I know it's way more verbose but it should help with HTML/CSS refactors.

If not though, it's okay.